src/components/TrackPain/NewPainEntry.js

```diff
@@ -30,6 +30,8 @@ import Snackbar from "@material-ui/core/Snackbar";
 import SnackbarContentWrapper from "../SnackbarContentComponent/SnackbarContentComponent";
 import { Redirect } from "react-router-dom";
 
+const colors = ["#4caf50", "#ffd95b", "#ff7043", "#c41c00"];
+
```

Comment:
I suggest using an objects for better readability
```js
const hexColorCodes = {
  green: "#4caf50",
  yellow: "#ffd95b",
  orange: "#ff7043",
  red: "#c41c00"
};
const intensityColorRange = {
  0: "green",
  1: "green",
  2: "green",
  3: "green",
  4: "yellow",
  5: "yellow",
  6: "yellow",
  7: "orange",
  8: "orange",
  9: "red",
  10: "red"
};
...
this.state = {
...
color: hexColorCodes.green
}
...
  setColor(painIntensityValue) {
    const color = intensityColorRange[painIntensityValue];

    return hexColorCodes[color];
  }
```

Reply:
I love it! Done!